tests/vmi_multus_test.go

```diff
@@ -1185,3 +1204,12 @@ func checkSriovEnabled(virtClient kubecli.KubevirtClient, sriovResourceName stri
 	}
 	return false
 }
+
+func getSRIOVInterfaceName(vmi *v1.VirtualMachineInstance) (error, string) {
+	for _, iface := range vmi.Spec.Domain.Devices.Interfaces {
+		if iface.SRIOV != nil {
+			return nil, iface.Name
```

Comment:
you need that the MAC will match
its right that you have just one in this specific test, but still